site/content/getting-started/_index.md

```diff
@@ -142,8 +142,8 @@ $ kubectl -n projectcontour get pods
 
 You should see the following:
 - 2 Contour pods with each status **Running** and 1/1 **Ready**  
-- 1+ Envoy pod(s), each with the status **Running** and 1/1 **Ready**  
- 
+- 1+ Envoy pod(s), with each the status **Running** and 2/2 **Ready**  
```

Comment:
I think the `each` can be left as-is but yep, should be 2 containers in the envoy pod

```suggestion
- 1+ Envoy pod(s), each with the status **Running** and 2/2 **Ready**  
```

Reply:
Thanks for quick review.
Good suggestion.
Rebased.